### PR TITLE
docs: Add DSP-META to docs

### PR DESCRIPTION
**Pull request by BalduinLandolt**

### CODE REVIEW

**Review by jnussbaum — APPROVED** [review]
Usually, we don't bump the submodules to the latest state. Rather, we wait until a deployment happens, and then we bump the submodules to the state **of the version that was deployed**. That means, the submodules should always be at the state defined in release.mk.

When you're on it, would you like to fix the wrong version of DSP-TOOLS in release.mk? It should be "v9.0.2", which is the latest clean release. (I think I once added the commit hash to make an informal after-deployment of the docs).

**Comment by BalduinLandolt** [comment]
> Usually, we don't bump the submodules to the latest state. Rather, we wait until a deployment happens, and then we bump the submodules to the state **of the version that was deployed**. That means, the submodules should always be at the state defined in release.mk.
> 
> When you're on it, would you like to fix the wrong version of DSP-TOOLS in release.mk? It should be "v9.0.2", which is the latest clean release. (I think I once added the commit hash to make an informal after-deployment of the docs).

Good thing you made me double-check! It was actually broken due to a wrong meta tag...

And, yes, I know that we normally don't update the submodules until the deployment, but I figured it doesn't matter as long as the docs don't get deployed. Should I revert?

**Comment by jnussbaum** [comment]
> > Usually, we don't bump the submodules to the latest state. Rather, we wait until a deployment happens, and then we bump the submodules to the state **of the version that was deployed**. That means, the submodules should always be at the state defined in release.mk.
> > When you're on it, would you like to fix the wrong version of DSP-TOOLS in release.mk? It should be "v9.0.2", which is the latest clean release. (I think I once added the commit hash to make an informal after-deployment of the docs).
> 
> Good thing you made me double-check! It was actually broken due to a wrong meta tag...
> 
> And, yes, I know that we normally don't update the submodules until the deployment, but I figured it doesn't matter as long as the docs don't get deployed. Should I revert?

No, you can leave it. Now, the submodules are in sync with release.mk, so that's okay.

**Comment by BalduinLandolt** [comment]
> > > Usually, we don't bump the submodules to the latest state. Rather, we wait until a deployment happens, and then we bump the submodules to the state **of the version that was deployed**. That means, the submodules should always be at the state defined in release.mk.
> > > When you're on it, would you like to fix the wrong version of DSP-TOOLS in release.mk? It should be "v9.0.2", which is the latest clean release. (I think I once added the commit hash to make an informal after-deployment of the docs).
> > 
> > 
> > Good thing you made me double-check! It was actually broken due to a wrong meta tag...
> > And, yes, I know that we normally don't update the submodules until the deployment, but I figured it doesn't matter as long as the docs don't get deployed. Should I revert?
> 
> No, you can leave it. Now, the submodules are in sync with release.mk, so that's okay.

But now the tests fail 🙈   
The last released version of meta does not have the mkdocs setup yet...

File: release.mk [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+# for DSP, API, APP, Ingest, see https://github.com/dasch-swiss/ops-deploy/blob/main/versions/RELEASE.json
 # for TOOLS, see https://github.com/dasch-swiss/dsp-tools/releases
+# for META, see https://github.com/dasch-swiss/dsp-meta/releases (NB: tags start with 'dsp-meta-')
 DSP := "2024.08.02"
 API := "v30.18.4"
 APP := "v11.18.2"
-TOOLS := "fdf58b9b8b7b244f3a99cc3629ffdb4e7d02d612"
+TOOLS := "v9.0.2"
 INGEST := "v0.12.3"
+META := "dsp-meta-v2.3.0"
```
Comment (mpro7) [inline release.mk:9]: This is a unfortunate setup :( but nothing related to this repo...

**Review by mpro7 — APPROVED** [review]

**Pull request merged**